src/lib/resourceFilters.js

```diff
@@ -0,0 +1,352 @@
+const fhirpath = require('fhirpath');
+// NOTE: See here: https://stackoverflow.com/questions/33704714/cant-require-default-export-value-in-babel-6-x
+const { ValueSetCodeChecker } = require('./ValueSetCodeChecker');
+
+const vsChecker = new ValueSetCodeChecker();
+
+// Patient
+// No reliable conformance requirements for mCODE Patients or US Core Patients: https://hl7.org/fhir/us/mcode/StructureDefinition-mcode-cancer-patient.html#conformance
+function getPatient(bundle) {
+  const metaProfiledResources = fhirpath.evaluate(
+    bundle,
+    "Bundle.entry.resource.where(meta.profile = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-cancer-patient')",
+  );
+  const constrainedResources = fhirpath.evaluate(bundle, 'Bundle.entry.resource.ofType(Patient)');
+  // NOTE: Being overly permissive is okay at this stage of the filter.
+  return metaProfiledResources || constrainedResources;
```

Comment:
This applies to every function in this file: Confusing Javascript truthyness strikes again! Empty arrays are truthy, so if there are no resources with `meta.profile`, the OR statement will see the empty array of `metaProfiledResources` as true and return an empty array. I tested this by removing every `meta.profile` from the `fullBundle.json` and saw that we get 0% coverage for everything.

Beyond the truthyness confusion, I think an OR is maybe the wrong choice anyway? For example, if someone had `meta.profile` on some but not all resources and we just returned the meta profiled ones, we would be missing all the non-meta.profile having resources, right?

My first thought was to do something like `return metaProfiledResources.concat(constrainedResources)`, but I think that would lead to duplicates. We might be able to get a combined list without dupes if we combined the two `fhirpath.evaluate` statements into one monster evaluate, maybe?